library/lib/build.js

```diff
@@ -330,12 +369,20 @@ module.exports = function build({ watch }) {
         templatePlugin(templateRenderers), // does work on .*.js
         mergeCssPlugin(),
         copyUnusedFilesPlugin(),
-        watch && hotCssReplacementPlugin()
+        watch && hotCssReplacementPlugin(),
+        webWorkerPlugin.handleWebWorkerImports,
```

Comment:
@EECOLOR  Ik weet niet precies wat dit doet maar hebben we dit nog nodig als we de import doen via een require?

Reply:
Ik verwacht dat we het wel nodig hebben. De dependency wordt vanuit webpack behandeld als `import`, de code wordt alleen nog niet uitgevoerd. Maar je kunt in die plugin even een `console.log` zetten om te checken of die nog steeds wordt uitgevoerd voor de `require` call.